tests/lib/cli_util_test.py

```diff
@@ -59,6 +59,10 @@ def test_parse_key_target(self):
         for spec, expected_parse in cases:
             self.assertEqual(cli_util.parse_key_target(spec), expected_parse)
 
+        usage_error_cases = [':', 'a:']
+        for spec in usage_error_cases:
```

Comment:
I would just inline this:

```suggestion
        for spec in [':', 'a:']:
```

Reply:
I was mainly trying to make the new test cases being consistent with the structure of the rest code in this test suite.